parser/index.ts

```diff
@@ -14,7 +13,7 @@ type Token = ['TEXT' | 'HTML', string];
 type StackItem = {tag: string; children: Array<StackItem | string>};
 
 function parseMarkdownToHTML(markdown: string): string {
-  const parser = ExpensiMark;
+  const parser = new ExpensiMark();
```

Comment:
I don't think it will work as Hermes doesn't still support classes and we don't transpile this code when generating Live Markdown JS bundle.
```suggestion
  const parser = ExpensiMark;
```

Reply:
@tomekzaw I updated the patch to export `ExpensiMark` as const.